runtime/tests/port/j9strTest.c

```diff
@@ -1534,40 +1534,45 @@ j9str_WinacpToMutf8(struct J9PortLibrary *portLibrary)
  * @param[in] buffer - pointer to the buffer intended for atoe_vsnprintf()
  * @param[in] bufferLength - the buffer length
  * @param[in] expectedOutput - The expected string
+ * @param[in] expectedCount - The expected response from atoe_vsnprintf()
  * @param[in] format - The string format
  *
  * @return TEST_PASS on success
  *         TEST_FAIL on error
  */
 static int
-printToBuffer(struct J9PortLibrary *portLibrary, BOOLEAN precision, char *buffer, size_t bufferLength, const char *expectedOutput, const char *format, ...)
+printToBuffer(struct J9PortLibrary *portLibrary, BOOLEAN precision, char *buffer, size_t bufferLength, const char *expectedOutput, int expectedCount, const char *format, ...)
 {
 	PORT_ACCESS_FROM_PORT(portLibrary);
-	int stringLength = 0;
+	int stringLength, expectedLength;
+	int actualCount;
```

Comment:
Please declare one local at a time and initialize each. In general, we follow the OMR coding guidelines [1].

[1] https://github.com/eclipse/omr/blob/master/CONTRIBUTING.md